README.md

```diff
@@ -1,28 +1,97 @@
 # govdoc-scanner
 
-This repository contains tools for processing Greek company documents (GEMI) to extract metadata and create historical timelines.
+## Project Overview
 
-**Install dependencies**  
- In the project root, run:
+In Greece, vital public company data is often locked in unstructured PDF files, making it challenging for citizens, researchers, and policymakers to access and analyze this information. The current state of these documents limits transparency and hinders efficient data use. The govdoc-scanner project seeks to bridge this gap by transforming these PDFs into a structured, searchable database, thereby democratizing access to important corporate information.
+
+The govdoc-scanner is an open-source tool designed to convert unstructured GEMI portal PDFs into a fully searchable database accessible via a REST API. It automates the extraction of metadata and document histories, making corporate information more accessible and useful for a wide range of users.
+
+## Current Functionality/Implementation
+
+The repository currently includes two main applications:
+
+- **doc-scanner**: Processes PDF/DOC/DOCX documents for a given GEMI company, extracting metadata and generating a contextual document history in JSON format.
+- **crawler**: Scrapes the GEMI portal to search for companies using advanced filters and downloads all available public documents for each company.
+- **orchestrator**: A script that automates the workflow of crawling and scanning for multiple GEMI IDs with progress tracking.
+
+All tools are implemented in Node.js and use a combination of CLI interfaces and automated scripts. The project uses Nx for monorepo management.
+
+## Usage Instructions
+
+### Requirements
+
+- **Node.js**: v18.x or newer (recommended: v20.x)
+  Check your versions with:
+
+```sh
+node --version
+```
+
+- **.env file**: You must create a `.env` file in the project root with your Gemini API key:
```

Comment:
What we can also do is commit in the repo a `.env.example` file which contains the key/value pairs which users should configure and instruct them here on copying the example env with `cp .env.example .env` and then updating the values with the appropriate entries.